lib/src/configuration.dart

```diff
@@ -252,20 +252,19 @@ abstract class Configuration implements Finalizable {
 /// that are persisted across runs.
 /// {@category Configuration}
 class LocalConfiguration extends Configuration {
-  LocalConfiguration._(
-    super.schemaObjects, {
-    this.initialDataCallback,
-    this.schemaVersion = 0,
-    super.fifoFilesFallbackPath,
-    super.path,
-    super.encryptionKey,
-    this.disableFormatUpgrade = false,
-    this.isReadOnly = false,
-    this.shouldCompactCallback,
-    this.migrationCallback,
-    super.maxNumberOfActiveVersions,
-    this.shouldDeleteIfMigrationNeeded = false
-  }) : super._();
+  LocalConfiguration._(super.schemaObjects,
+      {this.initialDataCallback,
```

Comment:
this seems not formatted correctly?

Reply:
The formatter started changing these for me as well - not sure what happened. I do prefer the old style, but it's also annoying to always revert back to it.

Reply:
It is already required to have a "," after the last parameter to order it as it was before.